tests/matrix_free/mass_operator_01.cc

```diff
@@ -116,7 +116,7 @@ void test ()
     mf_data.reinit (dof, constraints, quad, data);
   }
 
-  MatrixFreeOperators::MassOperator<dim,fe_degree, 1, number> mf;
+  MatrixFreeOperators::MassOperator<dim,fe_degree, fe_degree+1, 1, number> mf;
```

Comment:
sure, I am happy to change it. I was just keeping the original behaviour. Will update to `fe_degree+2`.